contracts/ibc-reflect/tests/integration.rs

```diff
@@ -66,7 +66,11 @@ fn fake_events(reflect_addr: &str) -> Vec<Event> {
             attr("module", "wasm"),
             attr("signer", MOCK_CONTRACT_ADDR),
             attr("code_id", "17"),
-            attr("contract_address", reflect_addr),
+            // We have to force this one to avoid the debug assertion against _
+            Attribute {
```

Comment:
I wonder if we should just create something like `Attribute::new_unchecked()`?

Reply:
That might make it too easy to bypass :wink: 

But I can add the helper if you wish.

Reply:
We could make it only available in debug builds. But it's just a thought. I'm good either way.

Reply:
Since this is only for very special cases in which wamd behaviour needs to be emulated, I would not create a constructor like that.

What we could do instead is having something like `mock_wasmd_attribute` that is only available in the testing module `cosmwasm_std::testing`.